mrp_bom_current_stock/reports/report_mrpcurrentstock.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="report_mrpbomstructure">
```

Comment:
IMO this template id should be something with 'current_stock' in it, to distinguish from the standard BOM structure report from odoo.